openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -62,6 +62,24 @@ def set_op_project(dbcon: AvalonMongoDB, project_id: str):
     dbcon.Session["AVALON_PROJECT"] = get_kitsu_project_name(project_id)
 
 
+def get_op_item_name_from_zou(
+        item: dict, parent_name: Optional[str]
+) -> str:
+    """Helper function to build OpenPype item name from zou item.
+
```

Comment:
```suggestion

Because OpenPype doesn't allow two entities to have the same name,
name is built with parents hierarchy `({episode}_){sequence}_{shot}`.

```

Reply:
Updated in https://github.com/kaamaurice/OpenPype/pull/165/commits/88b8ab8b06b4c656f5f367d6b3692cb69faa3514